src/bulk/common.ts

```diff
@@ -197,15 +197,17 @@ export class BulkWriteResult {
    * Create a new BulkWriteResult instance
    * @internal
    */
-  constructor(bulkResult: BulkResult) {
+  constructor(bulkResult: BulkResult, isOrdered: boolean) {
```

Comment:
Thoughts on the constructor just taking the `BulkOperationBase` as a parameter?  Both the bulkResult and the `isOrdered` property are stored on it anyways

Reply:
I think using BulkOperationBase as a parameter would pass a lot of extra information to the BulkWriteResult constructor, such as the internal state, which could potentially be confusing. What do you think?

Reply:
I don't feel strongly, feel free to ignore.  passing objects around is pretty common practice in JS.  especially when used with destructuring:

```javascript
constructor({ isOrdered, s: { bulkResult } }: BulkOperationBase ) {
   ... 
}
```

Reply:
Good to know!